app/src/main/java/org/mifos/mobile/ui/fragments/SavingAccountsTransactionFragment.kt

```diff
@@ -265,10 +264,10 @@ class SavingAccountsTransactionFragment : BaseFragment(), SavingAccountsTransact
         val llcheckBoxPeriod = dialogView?.findViewById<LinearLayout>(R.id.ll_row_checkbox)
         val checkBoxPeriod: AppCompatCheckBox? = dialogView?.findViewById(R.id.cb_select)
         val radioGroupFilter = dialogView?.findViewById<RadioGroup>(R.id.rg_date_filter)
+        val llstartEndDate = dialogView?.findViewById<LinearLayout>(R.id.ll_start_end_date)
```

Comment:
ll`S`, follow camelcase notation.

Reply:
Ohh sorry I copied format from checkbox period. I am updating it.

Reply:
@miPlodder review changes.